libs/text-splitters/langchain_text_splitters/character.py

```diff
@@ -577,3 +577,62 @@ def get_separators_for_language(language: Language) -> List[str]:
                 f"Language {language} is not supported! "
                 f"Please choose from {list(Language)}"
             )
+
+
+class SemanticTiktokenTextSplitter(TextSplitter):
+    """Splitting text that looks at characters."""
```

Comment:
@rgupta2508 Could you include more context both in the PR summary and in the class documentation?

For context, what's useful to know:

1) As a maintainer, what is this package that's being wrapped? Is there a link to it so we can check whether we want to recommend it to users.
2) As a user, how is this text splitter different from other text splitters? Why would a user chose to use this over something else?

Could you include in the doc-strings a URL to the implementation?